common/types.go

```diff
@@ -85,6 +98,18 @@ type ReadResult struct {
 	Done bool `json:"done,omitempty"`
 }
 
+// WriteResult is what's returned from writing data via the Write call.
+type WriteResult struct {
+	// ObjectId is the id of the written record.
+	ObjectId string
+	// Errors is list of error objects returned by the API.
+	Errors []interface{}
+	// Success is true if write succeeded.
+	Success bool
+	// RecordData is a JSON node containing data about the properties that were updated.
+	RecordData map[string]interface{} `json:"data"` //optional
```

Comment:
Same here, why is `json:"data"` a necessary annotation?

Reply:
if we marshal the `WriteResult` to json to return in the future (following the same pattern as ReadResult), we'd want this tag. I think we could/should actually add the json tag to the other fields.